### DISCUSSION TITLE
Initialize the model with select box

### DISCUSSION THREAD

**Discussion by MichaelMackus** [alpinejs/alpine#2956] [3. Feature ideas]
The model does not seem to be initialized to the selected element (using `x-model`). For example: https://alpinejs.dev/directives/model#single-select - you'll see that you have to actually change the value of the select to see the value update. I can always just initialize the model with `x-init` using the value of the select box, but I think this should be initialized like an input box is (when the value is set in HTML). I'm sure there are other priorities but this could help match how the rest of the x-model examples work.

**Comment by SimoTod** [comment]
Input box don't get initialised either (unless something has changed recently). The reasoning about that was to have a unique source of truth which is the component state so the value should be set on the relevant x-data properties.
I know some people suggested to have a modifier to force initialising from the element value but I don't think it's been implemented by anyone.
